frontend/src/lib/components/widgets/DataFrame/hooks/useColumnLoader.test.ts

```diff
@@ -111,12 +113,14 @@ describe("applyColumnConfig", () => {
     const column1 = applyColumnConfig(MOCK_COLUMNS[1], columnConfig)
     expect(column1.isEditable).toBe(true)
     expect(column1.width).toBe(COLUMN_WIDTH_MAPPING.small)
-    expect(column1.customType).toBe("text")
+    expect((column1.columnTypeOptions as any).type).toBe("text")
```

Comment:
Why is this being cast as any?

Reply:
This is just so that the linter is not running into an error. The alternative would be to somehow ignore the lining error here.

Reply:
Hm is there no type that we can cast it to? It feels better to add ignore the error here because I would never expect it to be cast as any but it's a super nit.

Reply:
Unfortunately, there isn't any good type. Since it's just a test, I will leave it this way here. If something is wrong here, the test will anyways fail.